plugins/Diagnostics/plugin.json

```diff
@@ -1,3 +1,6 @@
 {
-  "description": "Performs diagnostics to check that Piwik is installed and runs correctly."
+    "description": "Performs diagnostics to check that Piwik is installed and runs correctly.",
+    "require": {
+        "piwik": ">=2.16.0-b2"
```

Comment:
Should we always add such requires when a core plugin changes and requires features that will be shipped with a newer piwik version? Haven't seen that in another plugin that is shipped with core yet. But I guess it might make sense...